File: 231219/함수를 이용한 369 게임/369-games-using-functions.cpp
#include <iostream>
using namespace std;

bool IsTSN(int n) {
    return (n % 10) == 3 || (n % 10) == 6 || (n % 10) == 9 || (n / 10) == 3 || (n / 10) == 6 || (n / 10) == 9;
}

bool IsMagicNumber(int n) {
    return n % 3 == 0 || IsTSN(n);
}

int main() {
    int a, b, cnt = 0;
    cin >> a >> b;

    for (int i = a; i <= b; i++) {
        if(IsMagicNumber(i)) cnt++;
    }

    cout << cnt << endl;
    // 여기에 코드를 작성해주세요.
    return 0;
}

File: 250306/함수를 이용한 369 게임/369-games-using-functions.py
def is_magic_number(n):
    if n % 3 == 0 or is_include(n):
        return True
    return False


def is_include(n):
    while n > 0:
        ret = n % 10

        if ret == 3 or ret == 6 or ret == 9:
            return True
        
        n = n // 10
    

a, b = map(int, input().split())
cnt = 0

for i in range(a, b + 1):
    if is_magic_number(i):
        cnt += 1
print(cnt) 